js_modules/dagit/packages/core/src/runs/RunActionButtons.tsx

```diff
@@ -290,18 +291,25 @@ function usePipelineAvailabilityErrorForRun(
 
     if (matchType === 'snapshot-only') {
       // Only the snapshot ID matched, but not the repo.
+      const originRepoName = run.repositoryOrigin
+        ? repoAddressAsString(
+            buildRepoAddress(
+              run.repositoryOrigin.repositoryName,
+              run.repositoryOrigin.repositoryLocationName,
+            ),
+          )
+        : null;
+
       return {
         icon: 'warning',
         tooltip: (
           <Group direction="column" spacing={4}>
-            <div>{`The original run loaded "${run.pipelineName}" from a different repository.`}</div>
-            {run.repositoryOrigin ? (
+            <div>{`The original run loaded "${run.pipelineName}" from ${
+              originRepoName || 'a different code location'
```

Comment:
Another direct swap of "code location" instead of "repository", even though it might not currently be accurate for 1:N cases.